tools/format.ts

```diff
@@ -0,0 +1,21 @@
+console.log("rustfmt");
+
+await Deno.run({
+  cmd: ["rustfmt", "--check", "examples/dlint/main.rs"],
+  stdin: "null",
+  stdout: "null",
```

Comment:
Keep the output of subcommands - it's better to see output by default if something goes wrong